content/en/docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -115,6 +115,12 @@ the existing Pods based on the
 [priority](/docs/concepts/scheduling-eviction/pod-priority-preemption/#pod-priority)
 of the new Pod.
 
+{{< note >}}
+If it's important that the DaemonSet pod run on each node, it's often desirable
+to set the `.spec.template.spec.priorityClassName` of the DaemonSet to an existing [PriorityClass](/docs/concepts/scheduling-eviction/pod-priority-preemption/#priorityclass) 
+like `system-cluster-critical` to ensure that this eviction occurs.
```

Comment:
Avoid recommending this PriorityClass. If we name it here, people will set that without thinking about what they are doing.